audio-analysis/main.py

```diff
@@ -29,23 +31,22 @@ def analyzeVideoSound(url, tag):
     video.getAmplitudeAnalysis()
     video.getPitchAnalysis()
 
-    if (TESTING == False):
-        df1 = pd.DataFrame(video.word_list)
-        # We do not need two end_time_s_x and end_time_s_y columns :)
-        df2 = pd.DataFrame(video.amplitude_list).drop(columns=['end_time_s'])
-        join_condition = ['start_time_s', 'url']
-        # print(df1)
-        # print(df2)
-        df3 = dfHelper.customLeftJoin(df1, df2, join_condition)
-        if (df3.shape[0] == df1.shape[0]):
-            print('Join successful!')
-        else:
-            print('Join failed ... each word does not have an amplitude')
-        print(df3)
-        #Export video breakdown to CSV
-        df3.to_csv("data_export.csv", header=True)
-        #Export video breakdown to SQL
-        df3.to_sql("test_table", con=engine, if_exists='append')
+    df1 = pd.DataFrame(video.word_list)
+    # We do not need two end_time_s_x and end_time_s_y columns :)
+    df2 = pd.DataFrame(video.amplitude_list).drop(columns=['end_time_s'])
+    join_condition = ['start_time_s', 'url']
+    print(df1)
+    print(df2)
+    df3 = dfHelper.customLeftJoin(df1, df2, join_condition)
+    dfHelper.checkJoin(df3, df1, "Words - Amplitude")
```

Comment:
If checkJoin returns false do we want to stop? or keep joining?

Reply:
I also agreed with stopping ... the real fix will be another PR though. I like checkJoin a lot ... right now it is in the wrong place ... but lets leave it be. In the next 2-3 days will:
- make a test.py for all our unit tests using ye olde `unittest`
- use asserts the proper way in this test.py file to throw errors nicely.